schedule/yast/encryption/activate_encrypted_volume+import_users.yaml

```diff
@@ -31,3 +31,5 @@ schedule:
   - installation/reboot_after_installation
   - installation/grub_test
   - installation/first_boot
+test_data:
+  <<: !include test_data/yast/encryption/activate_encrypted_volume.yaml
```

Comment:
we are dragging here two more keys beside the one we need. to make it easy when we will be refactor this part it is better if we just repeat the test data in the test suite, it is more clear and visible.